grin/src/Reducer/PrimOps.hs

```diff
@@ -110,76 +108,76 @@ evalPrimOp name params args = case name of
   string x = pure . RT_Lit . LString $ x
 --  char x = pure . RT_Lit . LChar $ x
 
-  int_un_op retTy fn = case args of
+  int_un_op retTy fn args = case args of
     [RT_Lit (LInt64 a)] -> retTy $ fn a
-    _ -> error $ "invalid arguments: " ++ show params ++ " " ++ show args ++ " for " ++ unpackName name
+    _ -> error $ "invalid arguments: "++ show args
```

Comment:
We lose some info here, don't we?

Reply:
Yes. But I am OK with that as interpreters are mainly used in development / debugging phase.

Reply:
Isn't that a reason to keep that info? I mean, it would be nice to have as much info available in the debugging phase as possible. 

That being said, I don't think we need the parameters here.

Reply:
I felt it would have been hard to keep those parameters around. When we run into problems here we will improve the logging. That's what had happened before, and it will happen again. :)